src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/InputSkipIfLastAndSharedWithCase.java

```diff
@@ -0,0 +1,113 @@
+package com.puppycrawl.tools.checkstyle.checks.coding;
+
+
+public class InputSkipIfLastAndSharedWithCase
+{
+    void method(int i) {
+        switch (i) {
+            case 1:
+            default: // No violation with the new option is expected
+                break;
+            case 2:
+                break;
+        }
+
+        switch (i) {
+            case 1:
+            default: // No violation with the new option is expected
+            case 2:
+                break;
+            case 3:
+                break;
+        }
+
+        switch (i) {
+            default: // No violation with the new option is expected
```

Comment:
@rnveach 
i have shared diff report ,u could check it does not change original logic.
it should not show violation with new option because we can put default anywhere as long as it is surrounded by case.